tests/test_data/rendered_templates/default.rst

```diff
@@ -0,0 +1,60 @@
+.. _db:
+
+db
+--
+
+Created by H Wierstorf, C Geng, B E Abrougui
+
+============= ======================
+version       `1.0.0 <https://github.com/audeering/db/blob/main/CHANGELOG.md>`__
+license       `CC0-1.0 <https://creativecommons.org/publicdomain/zero/1.0/>`__
+source        https://github.com/audeering/audbcards
+usage         unrestricted
+languages     eng
+format        wav
+channel       1
+sampling rate 8000
+bit depth     16
+duration      0 days 00:00:04
+files         2
+repository    `data-local </tmp/pytest-of-cgeng/pytest-20/test_template_render0/repo/webapp/#/artifacts/browse/tree/General/data-local/db>`__
+published     2023-04-05 by cgeng
```

Comment:
Does this depend on who executes the code, or is `cgeng` hardcoded as author? If yes it might be better to use a more generic username to make this more obvious, e.g. `user`?

Reply:
- `cgeng` is the hardcoded string that comes from the template in `test_data/rendered_templates/default.rst`. 
-  the rendered content is determined by the `publication` property from `Dataset` . This uses the audfactory logic for other backends, but for local backends (that are probably only useful for testing), this assumes a `setup.cfg` to the present, and selects a random author from there. I am pretty sure that this is not a good alternative.  Prior version were even worse using the login name or the `$USER ` env. However these are unavailable in a CI.

Reply:
At a later stage we can write a custom backend for the tests that returns a fixed author and date.

For the repository I'm not sure if there will be a solution as this depends indeed on the underlying PC/runner.

For now, I would continue here ignoring those entries in the tests as you do already.